pennylane/devices/default_qubit_tf.py

```diff
@@ -157,7 +157,7 @@ class DefaultQubitTF(DefaultQubit):
     _stack = staticmethod(tf.stack)
 
     def __init__(self, wires, *, shots=1000, analytic=True):
-        super().__init__(wires, shots=shots, analytic=analytic)

```

Comment:
In `default.qubit.tf` and `default.qubit.autograd` so far we're not actually exposing `caching` to the user (hence no docstring). We put `caching=0` here to ensure that caching doesn't happen. The main motivation is that we don't support caching in backprop mode, which is the main use-case for these devices.

Alternatively, we could expose caching on these devices, e.g. just for forward passes, but I'm not sure if it will be used often.